UPGRADE-4.0.md

```diff
@@ -117,7 +117,7 @@ DependencyInjection
  * Using unsupported options to configure service aliases raises an exception.
 
  * Setting or unsetting a private service with the `Container::set()` method is
```

Comment:
`private` here does not make sense anymore either. Public services are forbidden too when not synthetic